File: 191-number-of-1-bits/number-of-1-bits.c
int hammingWeight(int n) {
    
    int remainder = 0 ;
    int setbit_cnt = 0;
    if (n == 0) {
        return setbit_cnt;
    }
        
    
    while (n > 0) {
        remainder = n % 2 ;
        if (remainder == 1)
           {
           setbit_cnt += 1 ;
           } 
        n = n / 2 ;
    } 
        
    

    return setbit_cnt ;

        
}

File: 191-number-of-1-bits/number-of-1-bits.py
class Solution:
    def hammingWeight(self, n: int) -> int:
        if n == 0: 
            return 0
        
        binary = ""
        set_bit_count = 0 
        while n > 0:
            remainder = n % 2 
            binary = str(remainder) + binary 
            if remainder == 1: 
                set_bit_count += 1 
            n = n//2 

        return set_bit_count 
        
